RangeTree/RangeTreeAsync.cs

```diff
@@ -1,279 +0,0 @@
-﻿using System;
```

Comment:
There exists no longer a version of the RangeTree that works with async? I'm not sure if anyone ever needed that.